lib/models/organisation-test.js

```diff
@@ -2,10 +2,71 @@ import chai, { expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { Error } from 'mongoose';
 import Organisation from './organisation';
+import User from './user';
+import Role from './role';
+import { createOrganisation, createUser } from './test-utils/testModelsHelper';
 
 chai.use(chaiAsPromised);
 
 describe('organisation model', () => {
+  afterEach(async () => {
+    await Role.deleteMany({});
+    await Organisation.deleteMany({});
+    await User.deleteMany({});
+  });
+
+  it('should properly create organisation', async () => {
+    const user = await createUser();
+    const organisation = await createOrganisation();
+
+    const userAfterCreateOrganisation = await User.findOne({
```

Comment:
I guess it will look better like so: `userAfterOrganisationCreated`

Reply:
Done!